Milestone 1: React capstone project - Metrics webapp

dqarias opened the pull request:
In this project we:

## Main Requirements

- Choose two API (the API should allow you to get numeric values based on a parameter)
    - [Rest Countries v3.1 API](https://restcountries.com/)
    - [Open Weather Map Air pollution API](https://openweathermap.org/api/air-pollution)
- Follow the original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).
- Built Home, Country and Pollution page.
- Built testing 

##Video

https://www.loom.com/share/3da955da93c44b9ba12f1325f24d77f9

dqarias committed:
Set up react project using CRA

dqarias committed:
Set up linters stylelint and eslint

dqarias committed:
Remove files from template to start from scratch

dqarias committed:
Select API for air pollution and countries

dqarias committed:
Install dependencies

dqarias committed:
Fetch data from rest countries API

dqarias committed:
Add reducer and thunks for fetching countries and Add continent acomponent

dqarias committed:
Set up router for countries route

dqarias committed:
Display countries on page and add route to get air pollution details

dqarias committed:
Update fetching data from air pollution api

dqarias committed:
Display pollution detail

dqarias committed:
Add image of each country's map on pollution detail

dqarias committed:
Add an country and continent's map

dqarias committed:
Add poprtypes for each component

dqarias committed:
Add a basic structure for styling

dqarias committed:
Add styling for continent card

dqarias committed:
Add a basic styling for pollution delait by country and add a gauge meter

dqarias committed:
Update styling on the whole project

dqarias committed:
Add functionality to search a country

dqarias committed:
Add styling on search country

dqarias committed:
Add functionality and styling for a visual loader

dqarias committed:
Add test for the project

dqarias committed:
Update stylin for desktop version

dqarias committed:
Fix linter errors

dqarias committed:
Update loader css

dqarias committed:
Update readme file

dqarias committed:
Fix linter errors

elisha2kyakpo1 reviewed (approved):
Hi 👋 @dqarias, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The PR title and description looks professional👍
- No linter errors👍
- Descriptive commit messages👍
- Your readme file is well documented 👍

## Optional suggestions

- [ ] There is a problem with the API you have used, I'm sure you know about this. Your deployed app doesn't work as expected.

![api](https://user-images.githubusercontent.com/53124548/181880268-ef81f016-ac11-4b1e-96fd-80ab68d55a94.png)


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

dqarias commented:
Hi @elisha2kyakpo1, thanks for your feedback and time, I didn't realize about that issue, I've just fixed it, the problem was the HTTP protocol of the air pollution API, I've changed it into HTTPS and it works properly  
- [live demo heroku](https://air-pollution-app-dino.herokuapp.com)
- [live demo netlify](https://air-pollution-app-dino.netlify.app/)
